Cargo.toml

```diff
@@ -11,19 +11,19 @@ members = [
 [profile.release]
 panic = 'unwind'
 
-[patch.crates-io]
-log = { git = "https://github.com/mesalock-linux/log-sgx" }
+#[patch.crates-io]
+#log = { git = "https://github.com/mesalock-linux/log-sgx" }
 
```

Comment:
Please remove the log patch.

Reply:
Yes I will - I left it here intentionally, because I really wasn't sure if I could remove these. But with you confirmation, I will do that.